.github/workflows/prod-docker.yml

```diff
@@ -0,0 +1,112 @@
+name: Production Aepp Faucet Pipeline
+
+on:
+  push:
+    branches:
+      - master
+    release:
+      types: [created]
+
+env:
+  ENV: "prd"
```

Comment:
are these prd/stg really named that way?

Reply:
Yes , they are named like that , thats what we decided with Dincho . It matches the gitops branches, etc..We can discussed it , if you have something in mind .

Reply:
not really, just unusual